### ISSUE TITLE
About training loss

### ISSUE THREAD

**Issue by whyandbecause** [prs-eth/Marigold#136]
Hi, thank you for your great work! I found there is only a mse loss for training, i'm wonder about why not use depth estimation loss function together for training? Thank you!

**Comment by nandometzger** [comment]
Hi

Finetuning the model with MSE loss on the latents has the advantage that you don't need to backpropagate through the decoder. This one of the reasons why we manage to finalize the model with only 1 24GB GPU. Several recent works showed that Marigold can also work as a one-step model, trained with depth supervision.
In particular also see: https://arxiv.org/abs/2409.11355

**Comment by whyandbecause** [comment]
Okay， thanks for your reply!
